user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -44,6 +44,7 @@
     private static final Logger log = LogManager.getLogger(KafkaUserModel.class.getName());
 
     public static final String KEY_PASSWORD = "password";
+    public static final String KEY_JAAS_CONFIG = "jaas_config";
```

Comment:
MAyve using `sasl.jaas.config` (the name of the related Kafka client property) would be better for this?

Reply:
I'm in agreement there. Better to closely mirror Kafka in this case.